### PR TITLE
Move last legacy API tests

### PR DESCRIPTION
**Pull request by nupplaphil**

### CODE REVIEW

**Comment by annando** [comment]
Why aren't no tests visible at this PR?

**Comment by nupplaphil** [comment]
I guess it was a bug .. I rebased it again and now it works.

File: tests/src/Module/BaseApiTest.php [hunk 1]
```diff
@@ -47,4 +48,14 @@ public function testWithWrongAuth()
 		);
 		*/
 	}
+
+	/**
+	 * Test the api_user() function.
```
Comment (annando) [inline tests/src/Module/BaseApiTest.php]: The function isn't named `api_user()` anymore.

File: tests/src/Util/DateTimeFormatTest.php [hunk 2]
```diff
@@ -77,4 +77,14 @@ public function testIsYearMonth(string $input, bool $assert)
 
 		self::assertEquals($assert, $dtFormat->isYearMonth($input));
 	}
+
+	/**
+	 * Test the api_date() function.
```
Comment (annando) [inline tests/src/Util/DateTimeFormatTest.php]: It's not really called `api_date()` anymore.

**Review by annando — COMMENTED** [review]
Just two small naming things.

**Comment by MrPetovan** [comment]
Thank you for your work!

**Pull request merged**